vietnamese/haloi/toi_la_nguoi_ha_loi_goc.md

```diff
@@ -0,0 +1 @@
+Quê gốc của tôi không phải ở Hà Nội, mà là Hải Phòng, tôi vừa nói dối đấy
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.